packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -30,6 +41,8 @@ export interface DataListProps extends Omit<React.HTMLProps<HTMLUListElement>, '
   selectedDataListItemId?: string;
   /** Flag indicating if DataList should have compact styling */
   isCompact?: boolean;
+  /** Specifies the grid breakpoints  */
+  gridBreakPoint?: 'none' | 'always' | 'sm' | 'md' | 'lg' | 'xl' | '2xl';
```

Comment:
Can we lowercase the `p` so it matches [OverflowMenu's spelling?](https://github.com/patternfly/patternfly-react/blob/164360d449dc64172e74159e276ed3fac2476593/packages/react-core/src/components/OverflowMenu/OverflowMenu.tsx#L24)